modules/web/js/ballerina/ast/statements/join-statment.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import _ from 'lodash';
+import Statement from './statement';
+
+class JoinStatement extends Statement {
+    constructor(args) {
+        super();
+        this.type = "JoinStatement";
+    }
```

Comment:
Can we set the _join_type variable in the constructor ? _.get(args, '_join_type').

Can we rename the variable to _joinType ?